src/layout/Footer.tsx

```diff
@@ -30,9 +31,18 @@ export const Footer = () => {
         bgcolor: 'info.light',
         display: 'grid',
         gridTemplateColumns: { xs: '1fr', md: '1fr 1fr 1fr' },
-        gap: '3rem',
```

Comment:
Bevisst at denne er tatt bort?
Fører til at noe av innholdet limes sammen på mobil
![bilde](https://github.com/user-attachments/assets/7ba6d867-e8f1-4a74-a57c-429a6f2eaf1d)

Reply:
Ja, men hadde ikke fått med meg glippen på mobil😬 Var for å unngå så stor gap på knappen. La inn en wrapper og `mb` på <LanguageSelector />